src/h5web/toolbar/controls/DomainSlider/DomainSlider.module.css

```diff
@@ -1,6 +1,7 @@
 .root {
   position: relative;
   display: flex;
+  justify-content: center;
```

Comment:
This is to keep the track aligned with the tooltip even if the parent div takes more width than the track

Reply:
The root `div` of the domain slider is not supposed to grow to fill the available space. Is this for the storybook? Maybe the issue comes from the decorator with `display: grid`. Do you need the decorator or can you remove it?

Reply:
In fact, I don't use the decorator. I am not sure why the root `div` takes the whole space in the Storybook...

Reply:
I removed this and used a new decorator `Center` to center the slider in the storybook.

Tell me what you think